docs_app/content/tutorial/7-flattening-operations.md

```diff
@@ -0,0 +1,132 @@
+
+### When NOT to use
+
+- Never use a concat strategy with extremely long-lived inner observables, or inner subscriptions that never complete. If it never completes, it cannot concatenate. Imagine another universe where you were trying to concat an array that was _infinitely long_ with any other array. You never get to where you're adding the contents of the second array because you cannot finish reading the first. Observables that never end cannot be concatenated.
+- When you have inner subscriptions you want to start right away. If this is the case, you'll probably want to use a [merge](#merge-merge-mergemap-mergeall-mergewith) or even a switch strategy.
+
+## Switching (switchMap, switchAll)
+
+Switching is a strategy that gets a decent amount of hype in RxJS tutorials. This comes from it's implicit cancellation behavior. This strategy will subscribe to one and only one inner observable at a time, but will subscribe to it as soon as it arrives. In order to ensure there is only one active inner subscription, it will unsubscribe from the previous inner subscription when the new one arrives. The switch strategy is the only strategy that will automatically unsubscribe from inner subscription to start the next one.
+
+It's called the "switch" strategy because it "switches" from whatever inner subscription it has to the next one it gets, as soon as it gets it.
+
+### When to use
+
+- Switch is particularly useful when dealing with situations where you want start the next subscription right away, and you do not care about the values of the previous one any more.
+- HTTP GETs involved with things like lookahead searches, or user-initiated HTTP GETS. It starts the request right away, and disregards the previous one, as you may not care about it.
+- Toggling between different observables. Let's say you had an animation loop and you wanted to turn it on and off, you could toggle between your animation loop observable and an empty observable using a switch strategy.
+
+### When NOT to use
+
+- Never use switchMap when the results that come back from your requests matter.
+- Do not use switchMap with HTTP requests where you care about the responses. In particular, HTTP methods like DELETE, PUT, and POST should not be used with a switching strategy if the responses are going to update the view. When in doubt, use the [concat strategy](#concat-concat-concatmap-concatall-concatwith).
+
+## Exhaust (exhaust, exhaustMap)
+
+Probably the least popular choice of our flattening strategies, but not because it's not useful. The exhaust strategy is sort of the "inverse of switching", in that it will still only allow one inner subscription at at time, but it will "exhaust" all values from that inner subscription, waiting for it to [complete](GL) before allowing another inner observable to be subscribed to. While there's one active inner subscription, all incoming inner subscriptions will be dropped.
```

Comment:
Not sure this first sentence is useful or adds to the explanation. Consider removing it.